xarray/conventions.py

```diff
@@ -293,10 +294,15 @@ def decode_cf_variable(name, var, concat_characters=True, mask_and_scale=True,
                       variables.CFScaleOffsetCoder()]:
             var = coder.decode(var, name=name)
 
+    enable_future_time_unit_decoding = OPTIONS['enable_future_time_unit_decoding']
```

Comment:
An suggestions for a shorter name here?

Reply:
I wasn't really worried about the linter but rather with the long name :smile:
But naming things **is** hard!